elyra/pipeline/processor_local.py

```diff
@@ -205,8 +205,13 @@ def process(self, operation: Operation):
                 filepath,
                 **additional_kwargs
             )
+        except papermill.PapermillExecutionError as pmee:
+            self.log.error(f'Internal error executing {file_name}: {str(pmee.ename)}' +
+                           f'{str(pmee.evalue)} in [{str(pmee.exec_count)}]')
+            raise RuntimeError(f'{str(pmee.ename)} {str(pmee.evalue)} in [{str(pmee.exec_count)}]') from pmee
```

Comment:
I find the `in [2]` portion of this confusing and regular users won't know what this means.  I suggest we drop this from the `RuntimeError` and even the log message.

The log will contain a traceback for this somewhere - I suspect when the web request throws - so that kind of information should be in the log.

Reply:
Will do! Makes sense if the normal user wouldn't get any useful info from it

Reply:
Perhaps if we use `notebook cell X?`

Reply:
@ptitzler - good idea, but I'd rather convey the cell index slightly differently...

`f'{str(pmee.ename)} {str(pmee.evalue)} executing cell {str(pmee.cell_index}]'`

which would yield...
```
Error processing operation load_data: FileNotFoundError [Errno 2] No such file or directory: 'data/file1.csv' executing cell 3
```
or
```
Error processing operation load_data: AssertionError executing cell 3
```
per examples above.  Does that sound okay?

(Note: I'm assuming the attribute name is `cell_index`.)

Reply:
The attribute is cell_index, correct!